app/components/Oauth.react.js

```
@@ -47,7 +47,7 @@ class Status extends Component {
                 });
             }
         }))
-        .catch(err => {
+        .catch(() => {
             this.setState({status: 'failure'});
```

let's add a `console.error(err)` here instead so that it doesn't get swallowed